cluster/gce/gci/configure-helper.sh

```diff
@@ -132,6 +135,9 @@ function main() {
 
     start-kube-apiserver
     start-kube-controller-manager
+    if [[ "${NETWORK_PROVIDER:-}" == "mizar" ]]; then
```

Comment:
If this only worked for scale-up, Please add  "${SCALEOUT_CLUSTER:-false}" == "false"